### ISSUE TITLE
Overfitting problem

### ISSUE THREAD

**Issue by ibozkurt79** [mtaruno/transformer-chatbot#38]
"Look at learning rate scheduling, after certain number of epochs, decrease learning rate.

Learning rate scheduling
Early stopping or reducing epochs
Dropout layers
Regularization
Improve distinctiveness between intent data"
how did you create H5 file you load ? Can you please share the steps you implemented above like rate scheduling in this notebook (intent classification) https://github.com/mtaruno/eve-bot/blob/master/3.%20Intent%20Classification%20with%20Keras.ipynb?
